src/screens/TransactionDetail.js

```diff
@@ -92,16 +97,53 @@ class TransactionDetail extends React.Component {
     }
   }
 
+  /**
+   * Method called when user clicked on 'See on explorer' link
+   *
+   * @param {Object} e Event for the click
+   */
+  goToExplorer = (e) => {
+    e.preventDefault();
+    const url = `https://explorer.hathor.network/transaction/${this.state.transaction.hash}`;
```

Comment:
I think the `BASE_URL` should be in the constants file.

Reply:
I'd call it `EXPLORER_BASE_URL` or something like that so it's clear this refers to the explorer.

Reply:
@obiyankenobi I agree.

Reply:
Done